pyiron_base/project/jobloader.py

```diff
@@ -53,11 +53,29 @@ def __getattr__(self, item):
     def __getitem__(self, item):
         return self.__getattr__(item)
 
-    def __call__(self, job_specifier, convert_to_object=None):
+    def __call__(self, job_specifier=None, db_entry=None, convert_to_object=None):
```

Comment:
From my perspective the `jobloader` is a user facing module and therefore should not use the `db_entry`, because I do not expect a user to define the dictionary of database entries.